components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/BPELServerImpl.java

```diff
@@ -896,6 +905,25 @@ public void onProcessStoreEvent(ProcessStoreEvent processStoreEvent) {
         }
     }
 
+    /**
+     * Function to check whether this nos node is manager node by checking registry read/write ability
+     *
+     * @return true if this is manager node, false otherwise
+     */
+    private boolean isManagerNode() {
+        try {
+            //get config registry of super tenant
+            Registry configRegistry = BPELServiceComponent.getRegistryService().getConfigSystemRegistry(-1234);
```

Comment:
Better to replace -1234 with MultitenantConstants.SUPER_TENANT_ID

Reply:
Fixed in 349c0e4